app/connectors/FinancialDataConnector.scala

```diff
@@ -55,21 +55,22 @@ class FinancialDataConnector @Inject()(http: HttpClient,
       }
   }
 
-  def getVatLiabilities(year: Int)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Seq[PaymentsHistoryModel]] = {
-    Future(Seq(
+  def getVatLiabilities(vrn: String, from: LocalDate, to: LocalDate)
+                       (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpGetResult[Seq[PaymentsHistoryModel]]] = {
+    Future.successful(Right(Seq(
       PaymentsHistoryModel(
-        taxPeriodFrom = LocalDate.parse(s"$year-01-01"),
-        taxPeriodTo   = LocalDate.parse(s"$year-02-01"),
+        taxPeriodFrom = LocalDate.parse(s"2018-01-01"),
+        taxPeriodTo   = LocalDate.parse(s"2018-02-01"),
```

Comment:
To make demonstration easier, you could make the year dynamic. To make it easy just get the year from the `from` LocalDate. Just to show that data changes between tabs.